cache/edge_lambdas/src/redirector.ts

```diff
@@ -38,14 +38,30 @@ const paramsAreSubset = (
 
 const filterParams = (
   params: URLSearchParams,
-  allow: Set<string>
+  allow: Set<string>,
+  modify?: {
+    [oldParamName: string]: string;
+  }[]
 ): URLSearchParams => {
   const filtered = new URLSearchParams();
+
   params.forEach((value, key) => {
+    // Check if param is in the forwardParams list
     if (allow.has(key)) {
-      filtered.append(key, value);
+      if (modify) {
+        // Check if param is also in the modifiedParams list
+        // If the params match, append the new param name to filtered
+        modify.find(modifiedParam => {
+          if (Object.keys(modifiedParam).find(k => k === key)) {
+            filtered.append(modifiedParam[key], value);
+          }
+        });
```

Comment:
Would strongly suggest not using `find` when the return value isn't used and/or when there are side effects (both true in this case).

Fix for this slightly depends on whether `modify` can be replaced with a single object, but regardless I think that 
```typescript
if (Object.keys(modifiedParam).find(k => k === key))
```
is exactly equivalent to
```typescript
if (modifiedParam[key])
```
or indeed
```typescript
if (key in modifiedParam)
```